internal/services/automation/automation_schedule_resource.go

```diff
@@ -7,6 +7,9 @@ import (
 	"strings"
 	"time"
 
+	// import time/tzdata to embed timezone information in the program
+	_ "time/tzdata"
```

Comment:
could we link to the github issue this is fixing so we don't unintentionally remove this?

Reply:
thanks for you suggestion, comment added. 🎉